test/e2e/run_networking_test.go

```diff
@@ -814,8 +839,44 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		pingTest("--net=private")
 	})
 
-	It("podman run check dnsname plugin", func() {
-		SkipUntilAardvark(podmanTest)
+	It("podman run check dnsname plugin with CNI", func() {
+		SkipIfNetavark(podmanTest)
+		pod := "testpod"
+		session := podmanTest.Podman([]string{"pod", "create", "--name", pod})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		net := "IntTest" + stringid.GenerateNonCryptoID()
+		session = podmanTest.Podman([]string{"network", "create", net})
+		session.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork(net)
+		Expect(session).Should(Exit(0))
+
+		pod2 := "testpod2"
+		session = podmanTest.Podman([]string{"pod", "create", "--network", net, "--name", pod2})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--name", "con1", "--network", net, ALPINE, "nslookup", "con1"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--name", "con2", "--pod", pod, "--network", net, ALPINE, "nslookup", "con2"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--name", "con3", "--pod", pod2, ALPINE, "nslookup", "con1"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(1))
+		Expect(session.ErrorToString()).To(ContainSubstring("can't resolve 'con1'"))
+
+		session = podmanTest.Podman([]string{"run", "--name", "con4", "--network", net, ALPINE, "nslookup", pod2 + ".dns.podman"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+	})
+
+	FIt("podman run check dnsname plugin with Netavark", func() {
```

Comment:
@flouthoc this is the second one that works. Other three fail.

Reply:
I guess this could be clubbed back in with the test for CNI to have a common version.

Reply:
This could be clubbed back with the copy for CNI test because there's no other change.